diff --git a/node_modules/@nuxt/content-theme-docs/src/index.js b/node_modules/@nuxt/content-theme-docs/src/index.js
index 1bbf230..6393b4d 100644
--- a/node_modules/@nuxt/content-theme-docs/src/index.js
+++ b/node_modules/@nuxt/content-theme-docs/src/index.js
@@ -45,6 +45,7 @@ function themeModule () {
     const slug = document.slug.replace(/^index/, '')
 
     document.to = `${dir}/${slug}`
+    document.draft = document.draft || false
   })
   // Extend `/` route
   hook('build:extendRoutes', (routes) => {
diff --git a/node_modules/@nuxt/content-theme-docs/src/pages/_.vue b/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
index 58bacc6..92a3074 100644
--- a/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
@@ -22,6 +22,7 @@
         </div>
 
         <NuxtContent :document="document" />
+        <Support />
       </article>
 
       <AppGithubLink :document="document" />
@@ -50,15 +51,16 @@ export default {
   },
   async asyncData ({ $content, store, app, params, error }) {
     const path = `/${app.i18n.locale}/${params.pathMatch || 'index'}`
-    const [document] = await $content({ deep: true }).where({ path }).fetch()
+    const [document] = await $content({ deep: true, draft: false }).where({ path }).fetch()
     if (!document) {
       return error({ statusCode: 404, message: 'Page not found' })
     }
 
     const [prev, next] = await $content(app.i18n.locale, { deep: true })
+      .where({ draft: false })
       .only(['title', 'slug', 'to'])
       .sortBy('position', 'asc')
-      .surround(document.slug, { before: 1, after: 1 })
+      .surround(document.path, { before: 1, after: 1 })
       .fetch()
 
     return {
diff --git a/node_modules/@nuxt/content-theme-docs/src/store/index.js b/node_modules/@nuxt/content-theme-docs/src/store/index.js
index 8d880e6..3867688 100644
--- a/node_modules/@nuxt/content-theme-docs/src/store/index.js
+++ b/node_modules/@nuxt/content-theme-docs/src/store/index.js
@@ -75,7 +75,7 @@ export const actions = {
     if (process.dev === false && state.categories[this.$i18n.locale]) {
       return
     }
-    const docs = await this.$content(this.$i18n.locale, { deep: true }).only(['title', 'menuTitle', 'category', 'slug', 'version', 'to']).sortBy('position', 'asc').fetch()
+    const docs = await this.$content(this.$i18n.locale, { deep: true }).where({ draft: false }).only(['title', 'menuTitle', 'category', 'slug', 'version', 'to']).sortBy('position', 'asc').fetch()
     if (state.releases.length > 0) {
       docs.push({ slug: 'releases', title: 'Releases', category: 'Community', to: '/releases' })
     }
